src/hooks/useSubscription.ts

```diff
@@ -45,13 +55,13 @@ export const useSubscription = <T = any, R = T, V = object>(
       })
     );
 
-    unsubscribe = teardown;
-  }, [request.key]);
+    unsubscribe.current = teardown;
+  }, [request, handler, client]);
```

Comment:
Same here, both `client` and `handler` are accessed in this `useCallback` and it makes sense to me that they're included.